src/Core/Domain/AbstractBulkCommandHandler.php

```diff
@@ -80,8 +79,9 @@ abstract protected function buildBulkException(array $coughtExceptions): BulkCom
 
     /**
      * @param mixed $id
+     * @param mixed $command
```

Comment:
```suggestion
     * @param mixed|null $command
```

Reply:
Fixed for better readability. Though I suppose `mixed` data type should includes `null` type as stated in doces https://www.php.net/manual/en/language.types.mixed.php